custom/icds_reports/migrations/sql_templates/update_tables49.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE agg_awc ADD COLUMN toilet_facility smallint;
+ALTER TABLE agg_awc ADD COLUMN type_toilet text;
+ALTER TABLE agg_awc ADD COLUMN preschool_kit_available smallint;
+ALTER TABLE agg_awc ADD COLUMN preschool_kit_usable smallint;
```

Comment:
do we think its worth putting some `CHECK` constraints on these columns? Similar to how `PositiveSmallInteger` puts a check constraint of > 0.

Reply:
I think these these columns are counts. So Should be valid in all condition i.e should range >0. Also there is no upper bound limit of these when on agg_level<5 So that way also there should be no limit.